src/terminal/select.c

```diff
@@ -371,17 +371,31 @@ void guac_terminal_select_end(guac_terminal* terminal) {
     /* Otherwise, copy multiple rows */
     else {
 
+        /* Get last char of the row */
+        guac_terminal_buffer_row* buffer_row = guac_terminal_buffer_get_row(terminal->buffer, start_row, 0);
```

Comment:
I'm a bit confused by the comment, here - wouldn't this particular call get the _first_ character of the given row (index 0 of the row), not the last char of the row?

Reply:
Yes, there was a small inconsistency since we retrieved a buffer row and not a char.
I pushed the correction.